types/net_fields.d.ts

```diff
@@ -306,6 +310,16 @@ export type NetFields = {
   'ActionSync': NetActionSyncFields;
   'StatusEffect': NetStatusEffectFields;
   'None': Fields;
-}
+};
+
+type Reverse<T> = {
```

Comment:
Can you leave a comment as to what `Reverse` is used for (so folks don't have to puzzle through TypeScript generics to figure it out).